business/istio_validations.go

```diff
@@ -61,7 +61,7 @@ func (in *IstioValidationsService) GetValidations(namespace, service string) (mo
 	var gatewaysPerNamespace [][]networking_v1alpha3.Gateway
 	var mtlsDetails kubernetes.MTLSDetails
 	var rbacDetails kubernetes.RBACDetails
-	var registryStatus []*kubernetes.RegistryStatus
+	var registryStatus []*kubernetes.RegistryService
```

Comment:
might be changing var name to 'registryService' ?

Reply:
+1